docs/Animations.md

```diff
@@ -7,24 +7,270 @@ permalink: docs/animations.html
+want because each can be customized:
+
+* `spring`: Simple single-spring physics model that matches Origami.
+ * `friction`: Controls "bounciness"/overshoot.  Default 7.
+ * `tension`: Controls speed.  Default 40.
+* `decay`: Starts with an initial velocity and gradually slows to a stop.
+ * `velocity`: Initial velocity.  Required.
+ * `deceleration`: Rate of decay.  Default 0.997.
+* `timing`: Maps time range to easing value.
+ * `duration`: Length of animation (milliseconds).  Default 500.
+ * `easing`: Easing function to define curve.  See `Easing` module for several
+ predefined functions.  iOS default is `Easing.inOut(Easing.ease)`.
+ * `delay`: Start the animation after delay (milliseconds).  Default 0.
+
+ Animations are started by calling `start`.  `start` takes a completion callback
+ that will be called when the animation is done.  If the animation is done
+ because it finished running normally, the completion callback will be invoked
+ with `{finished: true}`, but if the animation is done because `stop` was called
+ on it before it could finish (e.g. because it was interrupted by a gesture or
+ another animation), then it will receive `{finished: false}`.
+
+Animations can also be composed with `parallel`, `sequence`, `stagger`, and
+`delay`, each of which simply take an array of animations to execute and
+automatically calls start/stop as appropriate.  For example:
+
+```javascript
+Animated.sequence([            // spring to start and twirl after decay finishes
+  Animated.decay(position, {   // coast to a stop
+    velocity: {x: gestureState.vx, y: gestureState.vy}, // velocity from gesture release
+    deceleration: 0.997,
+  }),
+  Animated.parallel([          // after decay, in parallel:
+    Animated.spring(position, {
+      toValue: {x: 0, y: 0}    // return to start
+    }),
+    Animated.timing(twirl, {   // and twirl
+      toValue: 360,
+    }),
+  ]),
+]).start();                    // start the sequence group
+```
+
+By default, if one animation is stopped or interrupted, then all other
+animations in the group are also stopped.  Parallel has a `stopTogether` option
+that can be set to `false` to disable this.
+
+Another powerful part of the `Animated` API is the `interpolate` function.  It
+allows input ranges to map to different output ranges.  For example, a simple
+mapping to convert a 0-1 range to a 0-100 range would be
+
+```javascript
+value.interpolate({
+  inputRange: [0, 1],
+  outputRange: [0, 100],
+});
+```
+
+`interpolate` supports multiple range segments as well, which is handy for
+defining dead zones and other handy tricks.  For example, to get an negation
+relationship at -300 that goes to 0 at -100, then back up to 1 at 0, and then
+back down to zero at 100 followed by a dead-zone that remains at 0 for
+everything beyond that, you could do:
+
+```javascript
+value.interpolate({
+  inputRange: [-300, -100, 0, 100, 101],
+  outputRange: [300, 0, 1, 0, 0],
+});
+```
+
```

Comment:
Also probably useful here to talk about extrapolate options - this will be a problem that people will run into really quickly I think. Especially `extrapolate: 'clamp'`